app/facades/food_facade.rb

```diff
@@ -0,0 +1,19 @@
+class FoodFacade 
+   
+  class << self
+
+    def foods_by_keyword(food)
+      data = FoodService.foods_by_search(food)
+      @foods = data[:foods].map do |food|
+        Food.new(food)
+      end
+      @foods[0..9]
```

Comment:
If you add that additional param (see note in service), you won't need to use a range here